Stratis.Bitcoin/Features/MemoryPool/MempoolBehavior.cs

```diff
@@ -13,30 +13,86 @@
 
 namespace Stratis.Bitcoin.Features.MemoryPool
 {
+    /// <summary>
+    /// Node behavior for memory pool.
+    /// </summary>
     public class MempoolBehavior : NodeBehavior
     {
-        // Average delay between trickled inventory transmissions in seconds.
-        //  Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        #region Fields
+
+        /// <summary>
+        /// Average delay between trickled inventory transmissions in seconds.
+        /// Blocks and whitelisted receivers bypass this, outbound peers get half this delay. 
+        /// </summary>
         const int INVENTORY_BROADCAST_INTERVAL = 5;
-        // Maximum number of inventory items to send per transmission.
-        //  Limits the impact of low-fee transaction floods. 
+
+        /// <summary>
+        /// Maximum number of inventory items to send per transmission.
+        /// Limits the impact of low-fee transaction floods.
+        /// </summary>
         const int INVENTORY_BROADCAST_MAX = 7*INVENTORY_BROADCAST_INTERVAL;
 
+        /// <summary>
+        /// Memory pool validator injected dependency.
+        /// </summary>
         private readonly IMempoolValidator validator;
+
+        /// <summary>
+        /// Memory pool manager injected dependency.
+        /// </summary>
         private readonly MempoolManager manager;
```

Comment:
So, I was thinking about this "XYZ injected dependency" comment and I found out that I don't like it, sorry :-) 
It is true that it is XYZ injected dependency, but that is not very useful information. That tells me how the property was set (and I somehow do have that information from `readonly` keyword already). But mostly not really interested to learn how the property was set. Then it tells me that it is memory pool manager, which I +- get from the type.

I know this is hard in many cases, but it seems like you have a pattern here for all such fields even for those where it is not so hard to produce a better comment. So I would say that if a better comment is very hard to invent then OK, use this as a workaround, but otherwise, I would prefer a better comment.

What is better comment? I would be looking for an information telling me what is this in context of the class. So why do I need it here? What kind of feature it provides to me? 

I think this kind of comment is not good especially in combination with the trivial comment on `MempoolManager` class - so OK, let's say it is hard here to produce a better comment (I think it is not, but just to make the point), so I would expect that if I go to `MempoolManager` class I would see what it is, but there I read just "Memory pool manager". And at this point I'm lost and I'm forced to analyze the code.